packages/gas-fee-controller/src/GasFeeController.ts

```diff
@@ -190,12 +196,18 @@ export type FetchGasFeeEstimateOptions = {
  * @property gasFeeEstimates - Gas fee estimate data based on new EIP-1559 properties
  * @property estimatedGasFeeTimeBounds - Estimates representing the minimum and maximum
  */
-export type GasFeeState =
+export type GasFeeStateOld =
```

Comment:
```suggestion
export type ChainGasFeeState =
```

?

Reply:
Hmmm I agree `GasFeeStateOld` isn't great but I'm not sure I understand why `ChainGasFeeState` makes sense? Maybe `SingleChainGasFeeState`?

Reply:
Changed to `SingleChainGasFeeState` here: https://github.com/MetaMask/core/pull/1673/commits/9b53d8c39a968c10d089881357c9ce0f3dd63152

Reply:
Open to other suggestions though